src/platforms/hubspot/actions/emails/create.ts

```diff
@@ -0,0 +1,56 @@
+import { z } from 'zod';
+import { action } from '@/sdk';
+import client from '../../client';
+
+import { transformEmail } from '@/platforms/hubspot/actions/mappers';
+import { hubspotEmailDirectionSchema } from '../../schemas';
+
+export default action(
+  'emails-create',
+  {
+    operation: 'create',
+    resource: 'emails',
+    mutation: true,
+    schema: z.object({
+      fromEmail: z.string(),
+      toEmails: z.array(z.string()).nonempty(),
+      ccEmails: z.array(z.string()),
+      bccEmails: z.array(z.string()),
+      hsEmailHtml: z.string().optional(),
+      hsEmailText: z.string().optional(),
+      hsEmailDirection: hubspotEmailDirectionSchema.optional(),
+      hsEmailSubject: z.string().optional(),
+      hsEmailStatus: z.string().optional(),
+      hsTimestamp: z
+        .string()
+        .transform((v) => new Date(v))
+        .optional(),
```

Comment:
I think there is a datetime validator in zod which will throw a better error than this https://zod.dev/?id=iso-datetimes

Reply:
so it would be like ```z
        .string()
        .datetime()
        .transform((v) => new Date(v))
        .optional(),```